tests/unittests/test_discovery_utilities.py

```diff
@@ -204,10 +204,26 @@ def test_multiple_types_no_duplicates_returns_sorted(self):
 
         self.assertEqual(expected, actual)
 
+
+        actual = types_to_schema(['CURRENCY', 'TIME', 'INTEGER'], 'ga:testField')
+
+        expected = {'anyOf': [{'type': ['integer', 'null']},
+                              {'type': ['number', 'null']},
+                              {'type': ['string', 'null']}]}
+
+        self.assertEqual(expected, actual)
```

Comment:
Isn't this a duplicate of the test case above it?

Reply:
Yes, a relic of testing changes. Removed.